### DISCUSSION TITLE
Add Custom Properties to Register Page

### DISCUSSION THREAD

**Discussion by pengeron** [abpframework/abp#10001] [Q&A]
I followed the instructions on https://github.com/abpframework/abp/blob/dev/docs/en/Module-Entity-Extensions.md to add custom properties to the New User form. 

Any idea on how to add them to the Registration form ?

example, I added Address, City State and Zip

**Comment by maliming** [comment]
See https://support.abp.io/QA/Questions/1389/How-to-customize-register-page-MVC

**Reply by pengeron** [reply]
Sorry, forgot to mention I'm using Angular
